lib/processors/commons/new_file_string_processor.dart

```diff
@@ -31,4 +31,10 @@ class NewFileStringProcessor extends AbstractFileStringProcessor {
     String path,
     StringProcessor processor,
   ) : super(path, processor);
+
+  @override
+  void execute() {
+    file.createSync(recursive: true);
+    super.execute();
+  }
```

Comment:
I think that's not needed. The NewFileStringProcessor doesn't perform an existing check when loaded, so it shouldn't throw any exception.

I should write some dart doc in these basic classes :)

Reply:
It throws an exception when creating a file in a folder that does not exist, in this case `.vscode`.

```
Executing task ide:vscode
Unhandled exception:
FileSystemException: Cannot open file, path = '.vscode/launch.json' (OS Error: The system cannot find the path specified.
, errno = 3)
#0      _File.throwIfError (dart:io/file_impl.dart:645:7)
#1      _File.openSync (dart:io/file_impl.dart:489:5)
#2      _File.writeAsBytesSync (dart:io/file_impl.dart:614:31)
#3      _File.writeAsStringSync (dart:io/file_impl.dart:638:5)
#4      AbstractFileStringProcessor.execute (package:flutter_flavorizr/processors/commons/abstract_file_string_processor.dart:40:15)
#5      Processor.execute (package:flutter_flavorizr/processors/processor.dart:100:23)
<asynchronous suspension>
#6      execute (package:flutter_flavorizr/flutter_flavorizr.dart:43:13)
#7      main (file:///C:/Users/ryan.tauriainen/flutter_flavorizr/bin/flutter_flavorizr.dart:4:3)
#8      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:301:19)
#9      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)
pub finished with exit code 255
```

Reply:
If I create the folder before running, it processes it.

Reply:
I see, so I think we need to create the folder first, and then run the processors.

The ide:vscode command should be a QueueProcessor with a NewFolderProcessor (it doesn't exist, you will have to create it in the commons processor) and a NewFileProcessor with a nested VSCodeLaunchProcessor.